web/src/components/profileEdit/QuotaCard.tsx

```diff
@@ -16,82 +16,112 @@
+  quotaStorageSize: '',
+  quotaSnapshotSize: '',
+};
+
 const QuotaCard: React.FC<IQuotaCardProps> = (props) => {
   const {
     quotaDetails: {
       licensePlate = '',
       quotaSize = {
-        quotaCpuSize: '',
-        quotaMemorySize: '',
-        quotaStorageSize: '',
-        quotaSnapshotSize: '',
+        dev: NAMESPACE_DEFAULT_QUOTA,
+        test: NAMESPACE_DEFAULT_QUOTA,
+        tools: NAMESPACE_DEFAULT_QUOTA,
+        prod: NAMESPACE_DEFAULT_QUOTA,
       },
     },
+    href,
   } = props;
-
   const namespaceTexts = [
     ['Production', 'prod'],
     ['Test', 'test'],
     ['Development', 'dev'],
     ['Tools', 'tools'],
   ];
-  const specTexts: SpecTextInterface = {
-    CPU: quotaSize?.quotaCpuSize,
-    Memory: quotaSize?.quotaMemorySize,
-    Storage: quotaSize?.quotaStorageSize,
-    Snapshot: quotaSize?.quotaSnapshotSize,
-  };
 
   return (
     <Flex flexWrap="wrap">
-      {namespaceTexts.map((namespaceText: string[], index0: number) => (
-        <Aux key={index0}>
-          <Box width={1 / 2} px={2} mt={3}>
-            <Text as="h3">{namespaceText[0]} Namespace</Text>
-            <Text as="p" color={theme.colors.grey} fontSize={[1, 2, 2]} mt={1}>
-              {`${licensePlate}-${namespaceText[1]} namespace`}
-            </Text>
-          </Box>
-          <Box width={1 / 2} px={2} mt={3}>
-            <Text as="p" color={theme.colors.grey} fontSize={[2, 3, 3]} mt={1}>
-              CPU:{specTexts.CPU}
-            </Text>
-            <Text as="p" color={theme.colors.grey} fontSize={[2, 3, 3]} mt={1}>
-              Memory:{specTexts.Memory}
-            </Text>
-            <Text as="p" color={theme.colors.grey} fontSize={[2, 3, 3]} mt={1}>
-              Storage:{specTexts.Storage}
-            </Text>
-            <Text as="p" color={theme.colors.grey} fontSize={[2, 3, 3]} mt={1}>
-              Snapshot:{specTexts.Snapshot}
-            </Text>
-          </Box>
-        </Aux>
-      ))}
+      {namespaceTexts.map((namespaceText: string[], index0: number) => {
+        const index = namespaceText[1] as keyof typeof quotaSize;
+        return (
+          <Aux key={index0}>
+            <Box width={1 / 2} px={2} mt={3}>
+              <Flex>
+                <Text as="h3" marginX={2}>
+                  {namespaceText[0]} Namespace
+                </Text>
+                <RouterLink
+                  className="misc-class-m-dropdown-link"
+                  to={`${href}?namespace=${licensePlate}-${namespaceText[1]}`}
+                >
+                  <BaseIcon
+                    name="edit"
+                    color="primary"
+                    hover
+                    width={1.5}
+                    height={1.5}
+                    displayIcon={faPen}
+                  />
+                </RouterLink>
+              </Flex>
+              <Text as="p" color={theme.colors.grey} fontSize={[1, 2, 2]} mt={1}>
+                {`${licensePlate}-${namespaceText[1]} namespace`}
```

Comment:
seems like `${licensePlate}-${namespaceText[1]}` can be turned into a variable since its used more than once :)